image/cli/mascli/functions/install

```diff
@@ -16,6 +16,9 @@ Entitlement & Licensing (Required):
       --uds-email ${COLOR_YELLOW}UDS_CONTACT_EMAIL${TEXT_RESET}                        Contact e-mail address
       --uds-firstname ${COLOR_YELLOW}UDS_CONTACT_FIRSTNAME${TEXT_RESET}                Contact first name
       --uds-lastname ${COLOR_YELLOW}UDS_CONTACT_LASTNAME${TEXT_RESET}                  Contact last name
+      --dro-email ${COLOR_YELLOW}DRO_CONTACT_EMAIL${TEXT_RESET}                                Contact e-mail address
```

Comment:
Please let's just re-use the `--uds-xxx` parameters instead of making it harder for users.  Users don't care whether it's UDS or DRO (hell, we don't care really) ... just pass through uds-email/username/password to either UDS or DRO.

We can rename everything from UDS to DRO in a future major version change, but let's keep it so there's only one set of these.